app/services/action_handlers/pull_request_comment.rb

```diff
@@ -0,0 +1,25 @@
+module ActionHandlers
+  class PullRequestComment < ActionHandler
+    ACTIONS = %w[created edited deleted].freeze
+    private_constant :ACTIONS
+
+    private
+
+    def handleable?
+      ACTIONS.include?(@payload['action'])
+    end
+
+    def created
+      @entity.update!(body: @payload['comment']['body'])
+    end
+
+    def edited
+      @entity.update!(body: @payload['changes']['body'])
+      @entity.edited!
```

Comment:
What's it for?

Reply:
Is in case the comment is edited, to update the body text and change state.
Test file that would be included in next PR:
![Screen Shot 2021-07-27 at 14 27 53](https://user-images.githubusercontent.com/18470980/127200443-91e58407-3b74-4822-9c7d-78666ecab185.png)

But now I realize that an _edited_ state may not be necessary, at least for now. 🤔

Reply:
I get it. I asked you because I couldn't find the `edited!` function definition.